check/pytest

```diff
@@ -15,4 +17,20 @@
 cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd "$(git rev-parse --show-toplevel)"
 
-pytest $@
+PYTEST_ARGS=()
+ACTUALLY_QUIET=""
+for arg in $@; do
+    if [[ "${arg}" == "--actually_quiet" ]]; then
+        ACTUALLY_QUIET=1
+    else
+        PYTEST_ARGS+=("${arg}")
+    fi
+done
+
+if [ -z "${ACTUALLY_QUIET}" ]; then
+    pytest "${PYTEST_ARGS[@]}"
+else
+    # Filter out lines like "...F....x...      [ 42%]", with coloring.
+    pytest -q --color=yes "${PYTEST_ARGS[@]}" |
+        grep -Pv '^(.\[0m)?[\.FEsx]+(.\[36m)?\s+\[\s*\d+%\](.\[0m)?$'
```

Comment:
This seems kind of brittle but nothing bad will happen if pytest updates and it breaks.

Reply:
Yeah, I'm more worried about hiding real test output than I am about perfectly suppressing pytest's output.

Unfortunately it seems to have missed pytest's output on travis in the windows tests, so I need to tweak it at least a little.

Reply:
I have no idea how to fix the windows thing. For now I will defer it. https://github.com/quantumlib/Cirq/issues/1826